packages/vm/src/cache.rs

```diff
@@ -121,6 +91,7 @@ pub struct Cache<A: BackendApi, S: Storage, Q: Querier> {
     type_querier: PhantomData<Q>,
     /// To prevent concurrent access to `WasmerInstance::new`
     instantiation_lock: Mutex<()>,
+    wasm_limits: WasmLimits,
```

Comment:
What about just calling this "limits" instead of "wasm limits"? This allows us to easily expand the struct later on.

Reply:
The idea was to have different structs for the different areas of limits we have. This one for static validation, then maybe another one for read and deserialization limits (that one would also need to be passed into the `Environment`), etc.
And all those structs can then be combined in the `Config`.

Reply:
okay sounds good